showcase/src/app/demo/slider/slider.component.html

```diff
@@ -0,0 +1,133 @@
+<example-viewer [extensions]="['HTML', 'SCSS', 'TS']" 
+				filesPath="slider/simple-horizontal-slider"
+				title="SHOWCASE.DEMO.SLIDER.SIMPLE_HORIZONTAL_SLIDER">
+	<div>
+		<stark-slider [values]="simpleHorizontalSliderValues"
+					  sliderId="simple-horizontal-slider"
+					  [sliderConfig]="simpleHorizontalSliderConfig"
+					  [isDisabled]="!isSimpleHorizontalSliderEnabled"
+					  (changed)="onChangeSimpleHorizontalSlider($event)">
+		</stark-slider>
+		<div>
+			<mat-checkbox class="slider-enabled-field"
+						  (change)="$event ? toggleSimpleHorizontalSliderEnable() : null" 
+						  [checked]="isSimpleHorizontalSliderEnabled">
+				{{ 'SHOWCASE.COMMON.ENABLED' | translate }}
+			</mat-checkbox>
+			<mat-form-field class="slider-value-field">
+				<input matInput
+					   placeholder="{{ 'SHOWCASE.COMMON.VALUE' | translate }}"
+					   type="number"
+					   step="10"
+					   [(ngModel)]="valueForSimpleHorizontalSlider"
+					   [disabled]="!isSimpleHorizontalSliderEnabled"
+					   (change)="onChangeValueForSimpleHorizontalSlider($event)">
+			</mat-form-field>
+		</div>
+	</div>
+</example-viewer>
+
+<example-viewer [extensions]="['HTML', 'SCSS', 'TS']"
+				filesPath="slider/horizontal-range-slider"
+				title="SHOWCASE.DEMO.SLIDER.HORIZONTAL_RANGE_SLIDER">
```

Comment:
Same here, this binding has been changed after your PR.
Could you please adapt this for `exampleTitle="SHOWCASE.DEMO.SLIDER.HORIZONTAL_RANGE_SLIDER"` ?